src/ldp/LDPCommunication.ts

```diff
@@ -11,11 +11,12 @@ export class LDPCommunication implements Communication {
     public constructor() {
     }
 
+
     public async get(resourceIdentifier: string, headers?: Headers): Promise<Response> {
         headers = headers ? headers : new Headers({'Accept': 'text/turtle'})
         return await fetch(resourceIdentifier, {
             method: 'GET',
-            headers
+            headers: headers
```

Comment:
I don't see the benefit of this change

Reply:
Consistency with all the other functions and in my personal opinion Im not a huge fan of that syntax, it's a bit cryptic imo.